raiden/channel.py

```diff
@@ -599,8 +600,8 @@ def channel_closed(self, block_number):
     def blockalarm_for_settle(self, block_number):
         def _settle():
             for _ in range(3):
-                # do not call settle if already settled, the event polling
-                # might be lagging behind.
+                # do not call settle if already settled, the settle_event might
+                # not be set if the LogListener is lagging behind.
```

Comment:
Cool. I didn't know the `--amend` command before now. The other times you've said amend the commit, I thought it was meant literally.